docs_src/V3TopLevelMigration.md

```diff
@@ -90,6 +90,11 @@ The following sections describe what you need to be aware for the different serv
 
 The Event/Reading data stored by Core Data is considered transient and of little value once it has become old. The V3 versions of these data collections have minimal changes from their V2 counter parts. 
 
+#### API Change
+- In order to identify the device service generating the new event, 
+POST Event REST endpoint will become `/event/{serviceName}/{profileName}/{deviceName}/{sourceName}`
```

Comment:
```suggestion
- Add Event
    POST endpoint is now  `/event/{serviceName}/{profileName}/{deviceName}/{sourceName}`. This is so that the metadata contains the device service which generated the new event
```

Reply:
Add reference to swagger: https://docs.edgexfoundry.org/{version}/api/core/Ch-APICoreData/

Reply:
updated